src/styles/blocks/watch.scss

```
@@ -0,0 +1,76 @@
+.watch {
+  position: relative;
+
+  @include circle($watch-size);
+  @include center-block;
+
+  background-color: $bgc;
+  border-radius: 50%;
+  border: 15px double $yellowColor;
+
+  &__center {
+    @include circle($center-size);
+
+    background-color: #001064;
```

You created a variables for the colors, but not for all 🥺 
```suggestion
    background-color: $c-someColor;
```